### PR TITLE
[FR] Add `Help centre/Upgrading to lazer`

### PR DESCRIPTION
**Pull request by Pisapou**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Help_centre/Upgrading_to_lazer/fr.md [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+  - client du jeu
+---
+
+# Mise à jour vers lazer
+
+osu!(lazer) est la prochaine mise à jour majeure du jeu. C'est l'aboutissement de plusieurs années de travail en coulisses pour réimplémenter minutieusement le jeu.
+
+L'objectif final est que cette version soit publiée comme une mise à jour qui remplacera complètement la version stable existante du jeu, mais il y a encore du travail à faire avant que cela ne soit réalisable. En fin de compte, **les joueurs** décideront du moment où cela se produira et nous continuerons à supporter la version précédente jusqu'à ce que tous les utilisateurs aient migré.
+
+"lazer " est un nom de code et sera éventuellement abandonné lorsqu'il deviendra la version principale du jeu. Dans le reste de ce document, osu!(lazer) sera appelé "lazer" et osu!(stable) sera appelé "stable" pour plus de simplicité.
```
Comment (Walavouchey) [inline wiki/Help_centre/Upgrading_to_lazer/fr.md:13]: are these smart quotes (`"` and `"`) already a thing in other translations? the english versions always use `"`

```suggestion
"lazer" est un nom de code et sera éventuellement abandonné lorsqu'il deviendra la version principale du jeu. Dans le reste de ce document, osu!(lazer) sera appelé "lazer" et osu!(stable) sera appelé "stable" pour plus de simplicité.
```

File: wiki/Help_centre/Upgrading_to_lazer/fr.md [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+
+## Comparaison des fonctionnalités
+
+Ce qui suit est une liste complète de l'**état actuel** de lazer par rapport à stable. Notez qu'il n'est pas définitif — l'objectif final est d'implémenter toutes les fonctionnalités qui intéressent les joueurs au fil du temps.
+
+### Compatibilité et performance
+
+| Fonctionnalité | stable | lazer |
+| :-- | :-- | :-- |
+| Windows 8.0 et moins | ![Yes][true] | ![No][false] |
```
Comment (Walavouchey) [inline wiki/Help_centre/Upgrading_to_lazer/fr.md:23]: etc

```suggestion
| Windows 8.0 et moins | ![Oui][true] | ![Non][false] |
```

File: wiki/Help_centre/Upgrading_to_lazer/fr.md [hunk 3]
```diff
@@ -0,0 +1,289 @@
...
+### Systèmes en ligne
+
+| Fonctionnalité | stable | lazer |
+| :-- | :-- | :-- |
+| Soumission des scores | ![Oui][true] | ![Partiellement][partial][^score-reset-balance] |
+| Tableaux de classement de beatmap | ![Oui][true] | ![Partiellement][partial][^score-reset-isolated] |
+| Statistiques du profil | ![Oui][true] | ![Oui][true] |
+| Médailles | ![Oui][true] | ![Non][false] |
+| Points de performance | ![Oui][true] | ![Partiellement][partial][^score-reset-isolated] |
+| Chat en temps réel | ![ParPartiellementtial][partial][^stable-chat] | ![Oui][true] |
```
Comment (Pisapou) [inline wiki/Help_centre/Upgrading_to_lazer/fr.md:66]:
```suggestion
| Chat en temps réel | ![Partiellementtial][partial][^stable-chat] | ![Oui][true] |
```

Comment (Pisapou) [inline wiki/Help_centre/Upgrading_to_lazer/fr.md:66]:
```suggestion
| Chat en temps réel | ![Partiellement][partial][^stable-chat] | ![Oui][true] |
```

File: wiki/Help_centre/Upgrading_to_lazer/fr.md [hunk 4]
```diff
@@ -0,0 +1,289 @@
...
+
+Enfin, contrairement à la dernière itération, nous consacrons du temps et de la diligence pour nous assurer que la base de code nous servira à l'avenir. Nous avons fait le travail de base pour permettre aux nouvelles fonctionnalités d'être mises en ligne à une vitesse fulgurante à l'avenir. Il s'agira notamment de nouveaux composants d'UI, de nouvelles façons de personnaliser le jeu, de nouveaux systèmes multijoueurs, sans oublier la possibilité de charger et de jouer tous vos beatmaps existants dans des modes de jeu entièrement nouveaux (a.k.a. les rulesets) !
+
+#### Quelle est la prochaine étape ?
+
+Nous avons un énorme arriéré de fonctionnalités et d'améliorations demandées par les utilisateurs, que nous allons continuer à mettre en place à la vitesse de la lumière. Pour ceux qui nous ont rejoints récemment et qui n'ont pas connu l'élan du développement d'osu!, préparez-vous à être surpris.
+
+#### Comment accéder à mon dossier songs ?
```
Comment (Ganondorf-1) [inline wiki/Help_centre/Upgrading_to_lazer/fr.md:230]:
```suggestion
#### Comment accéder à mon dossier Songs ?
```

File: wiki/Help_centre/Upgrading_to_lazer/fr.md [hunk 5]
```diff
@@ -0,0 +1,289 @@
...
+
+Il n'y a pas de dossier songs dans lazer! Cela nous permet de faire des choses sympas comme ne pas avoir à presser `F5` lors de la sélection des beatmaps pour les rafraîchir (parce que les beatmaps sont toujours dans un bon état) et réduire l'espace disque utilisé par les beatmaps de 20-40%. Vous pouvez en savoir plus sur [la façon dont lazer stocke les fichiers](/wiki/Client/Release_stream/Lazer/File_storage).
```
Comment (Ganondorf-1) [inline wiki/Help_centre/Upgrading_to_lazer/fr.md:232]:
```suggestion
Il n'y a pas de dossier Songs dans lazer ! Cela nous permet de faire des choses sympas comme ne pas avoir à presser `F5` lors de la sélection des beatmaps pour les rafraîchir (parce que les beatmaps sont toujours dans un bon état) et réduire l'espace disque utilisé par les beatmaps de 20-40%. Vous pouvez en savoir plus sur [la façon dont lazer stocke les fichiers](/wiki/Client/Release_stream/Lazer/File_storage).
```

File: wiki/Help_centre/Upgrading_to_lazer/fr.md [hunk 6]
```diff
@@ -0,0 +1,289 @@
...
+
+Si vous devez apporter des modifications à une beatmap, veuillez utiliser l'éditeur. À l'avenir, nous introduirons un mode dans l'éditeur qui rendra le dossier d'un beatmap temporairement accessible pour une édition externe. Cela vous permettra d'utiliser des outils externes sur une beatmap pendant le processus de création.
```
Comment (Ganondorf-1) [inline wiki/Help_centre/Upgrading_to_lazer/fr.md:234]:
```suggestion
Si vous devez apporter des modifications à une beatmap, veuillez utiliser l'éditeur. À l'avenir, nous introduirons un mode dans l'éditeur qui rendra le dossier d'une beatmap temporairement accessible pour une édition externe. Cela vous permettra d'utiliser des outils externes sur une beatmap pendant le processus de création.
```

File: wiki/Help_centre/Upgrading_to_lazer/fr.md [hunk 7]
```diff
@@ -0,0 +1,289 @@
...
+
+De la même manière que d'habitude.
+
+#### Où sont les micro-transactions ?
+
+Vous pensez probablement à un autre jeu.
+
+### Notes
+
+[^wine]: En utilisant du vin.
```
Comment (Ganondorf-1) [inline wiki/Help_centre/Upgrading_to_lazer/fr.md:260]:
```suggestion
[^wine]: En utilisant Wine.
```

File: wiki/Help_centre/Upgrading_to_lazer/fr.md [hunk 8]
```diff
@@ -0,0 +1,289 @@
...
+[^compatibility-mode]: DirectX via le mode de compatibilité.
+[^coming-soon]: A venir prochainement.
```
Comment (Ganondorf-1) [inline wiki/Help_centre/Upgrading_to_lazer/fr.md:262]:
```suggestion
[^coming-soon]: À venir prochainement.
```

File: wiki/Help_centre/Upgrading_to_lazer/fr.md [hunk 9]
```diff
@@ -0,0 +1,289 @@
...
+[^dll]: Manuellement via les fichiers `.dll`.
+[^share-files]: Les beatmaps et les skins partageront les fichiers et économiseront de l'espace disque.
+[^gameplay-only]: Gameplay uniquement.
+[^online]: Par le biais d'une recherche en ligne.
+[^normalisation]: Cela permet d'amener les combo colours personnalisée de beatmap au même niveau de luminosité.
```
Comment (Ganondorf-1) [inline wiki/Help_centre/Upgrading_to_lazer/fr.md:267]:
```suggestion
[^normalisation]: Cela permet d'amener les combo colours personnalisées de beatmap au même niveau de luminosité.
```

File: wiki/Help_centre/Upgrading_to_lazer/fr.md [hunk 10]
```diff
@@ -0,0 +1,289 @@
...
+[^hold-for-hud]: Maintenez `Ctrl` pour voir le HUD momentanément pendant qu'il est caché.
+[^offset-calibration-stable]: Réglable manuellement par la configuration des touches.
+[^offset-calibration-lazer]: Lorsque vous réessayez une beatmap, vous pouvez calibrer le décalage audio en fonction de votre dernière partie.
+[^can-disable]: Peut être désactivé.
+[^note-lock]: Existe toujours, mais ne devrait pas interférer.
+[^score-reset-balance]: Les scores seront remis à zéro pour assurer l'équilibre.
+[^score-reset-isolated]: Les scores seront réinitialisés, actuellement isolés des scores stables.
+[^online-content]: Accès natif à la plupart des contenus en ligne.
+[^direct-supporter]: Uniquement via osu!direct, réservé aux osu!supporter.
```
Comment (Ganondorf-1) [inline wiki/Help_centre/Upgrading_to_lazer/fr.md:276]:
```suggestion
[^direct-supporter]: Uniquement via osu!direct, réservé aux osu!supporters.
```

**Review by Ganondorf-1 — CHANGES_REQUESTED** [review]

**Comment by The-Last-Cookie** [comment]
Please don't forget to apply the changes from #8548.

File: wiki/Help_centre/Upgrading_to_lazer/fr.md [hunk 11]
```diff
@@ -0,0 +1,289 @@
...
+| Support de skin | ![Oui][true] | ![Partiellement][partial][^gameplay-only] |
+| Modes de regroupement par musique | ![Oui][true] | ![Non][false] |
+| Skin en jeu / Modification de la mise en page de l'UI | ![Non][false] | ![Oui][true] |
+
+### Gameplay et skinning
+
+| Fonctionnalité | stable | lazer |
+| :-- | :-- | :-- |
+| Affichage précis des points de performance | ![Partiellement][partial][^online] | ![Oui][true] |
+| Préréglages de mods | ![Non][false] | ![Oui][true] |
```
Comment (Pisapou) [inline wiki/Help_centre/Upgrading_to_lazer/fr.md:47]:
```suggestion
| Ajustement de la difficulté de mods | ![Non][false] | ![Oui][true][^difficulty-adjust] |
| Préréglages de mods | ![Non][false] | ![Oui][true] |
```

File: wiki/Help_centre/Upgrading_to_lazer/fr.md [hunk 12]
```diff
@@ -0,0 +1,289 @@
...
+
+### Performance
+
+#### Pourquoi ne puis-je pas jouer avec un nombre illimité de FPS ?
+
+Au-delà d'un certain seuil, il n'y a aucune raison de fonctionner à des fréquences d'images plus élevées. Lazer utilise diverses nouvelles technologies pour s'assurer que la latence la plus faible possible soit atteinte sans nécessiter des fréquences d'images élevées. Cela continuera à s'améliorer à l'avenir, car il nous reste encore quelques améliorations à mettre en œuvre.
+
+Lazer prend en compte l'entrée à 1000 Hz indépendamment du limiteur de FPS, c'est pourquoi le réglage maximal du limiteur s'arrêtera également à 1000 FPS.
+
+Si vous êtes curieux de savoir comment cela affecte la latence d'entrée et de tester votre propre perception, veuillez exécuter le "certificateur de latence" intégré en bas des paramètres.
```
Comment (Pisapou) [inline wiki/Help_centre/Upgrading_to_lazer/fr.md:194]:
```suggestion
Si vous êtes curieux de savoir comment cela affecte la latence d'entrée et de tester votre propre perception, veuillez exécuter le "certificateur de latence" intégré en bas des paramètres.

Vous pouvez également [lire ce document technique] (https://github.com/ppy/osu/wiki/Latency-and-unlimited-frame-rates) qui explique la voie que nous empruntons et le raisonnement qu'elle implique.
```

File: wiki/Help_centre/Upgrading_to_lazer/fr.md [hunk 13]
```diff
@@ -0,0 +1,289 @@
...
+
+Nous allons probablement rétablir cette fonctionnalité à l'avenir en raison de la demande populaire.
+
+### Performance
+
+#### Pourquoi ne puis-je pas jouer avec un nombre illimité de FPS ?
+
+Au-delà d'un certain seuil, il n'y a aucune raison de fonctionner à des fréquences d'images plus élevées. Lazer utilise diverses nouvelles technologies pour s'assurer que la latence la plus faible possible soit atteinte sans nécessiter des fréquences d'images élevées. Cela continuera à s'améliorer à l'avenir, car il nous reste encore quelques améliorations à mettre en œuvre.
+
+Lazer prend en compte l'entrée à 1000 Hz indépendamment du limiteur de FPS, c'est pourquoi le réglage maximal du limiteur s'arrêtera également à 1000 FPS.
```
Comment (Pisapou) [inline wiki/Help_centre/Upgrading_to_lazer/fr.md:192]:
```suggestion
Lazer prend en compte l'entrée à 1 000 Hz indépendamment du limiteur de FPS, c'est pourquoi le réglage maximal du limiteur s'arrêtera également à 1 000 FPS.
```

File: wiki/Help_centre/Upgrading_to_lazer/fr.md [hunk 14]
```diff
@@ -0,0 +1,289 @@
...
+
+Si vous êtes curieux de savoir comment cela affecte la latence d'entrée et de tester votre propre perception, veuillez exécuter le "certificateur de latence" intégré en bas des paramètres.
+
+#### Si l'entrée n'est prise en compte qu'à 1000 Hz, qu'en est-il de ma souris gaming à 8000 Hz ?
+
+Le système d'exploitation continuera à prendre en compte à un taux plus élevé, bien que les avantages s'avèrent être négligeables. Le rafraîchissement à des taux aussi élevés peut avoir des surcharges imprévues, et nous recommandons de limiter les périphériques à 1000 Hz pour la stabilité du système.
```
Comment (Pisapou) [inline wiki/Help_centre/Upgrading_to_lazer/fr.md:198]:
```suggestion
Le système d'exploitation continuera à prendre en compte à un taux plus élevé, bien que les avantages s'avèrent être négligeables. Le rafraîchissement à des taux aussi élevés peut avoir des surcharges imprévues, et nous recommandons de limiter les périphériques à 1 000 Hz pour la stabilité du système.
```

File: wiki/Help_centre/Upgrading_to_lazer/fr.md [hunk 15]
```diff
@@ -0,0 +1,289 @@
...
+[^direct-supporter]: Uniquement via osu!direct, réservé aux osu!supporters.
+[^multi-room-max]: 16 joueurs max.
+[^map-only]: Map seulement.
+[^all-files]: Tous les fichiers.
+[^editor-precise-rotation]: Il manque une rotation précise des angles.
+[^incompatibilities]: Certaines fonctionnalités de l'éditeur provoquent une lecture incorrecte des beatmaps dans la version stable — sera corrigé bientôt.
+[^stable-chat]: Les messages peuvent prendre jusqu'à 15 secondes pour arriver.
+[^countdown-timers-stable]: Définir un compte à rebours à l'aide d'une commande, pas de démarrage automatique.
+[^countdown-timers-lazer]: Définissez un compte à rebours depuis l'UI du jeu pour lancer automatiquement le match.
+[^queue-modes]: Activez cette option pour permettre à n'importe qui dans un lobby de mettre en file d'attente de nouvelles beatmaps, alias "rotation de l'hôte".
```
Comment (Pisapou) [inline wiki/Help_centre/Upgrading_to_lazer/fr.md:285]:
```suggestion
[^queue-modes]: Activez cette option pour permettre à n'importe qui dans un lobby de mettre en file d'attente de nouvelles beatmaps, alias "rotation de l'hôte".
[^difficulty-adjust]: Changez le CS/AR/OD/HP d'une beatmap directement depuis la sélection des beatmaps.
```

File: wiki/Help_centre/Upgrading_to_lazer/fr.md [hunk 16]
```diff
@@ -0,0 +1,293 @@
...
+| Support de skin | ![Oui][true] | ![Partiellement][partial][^gameplay-only] |
+| Modes de regroupement par musique | ![Oui][true] | ![Non][false] |
+| Skin en jeu / Modification de la mise en page de l'UI | ![Non][false] | ![Oui][true] |
+
+### Gameplay et skinning
+
+| Fonctionnalité | stable | lazer |
+| :-- | :-- | :-- |
+| Affichage précis des points de performance | ![Partiellement][partial][^online] | ![Oui][true] |
+| Ajustement de la difficulté de mods | ![Non][false] | ![Oui][true][^difficulty-adjust] |
```
Comment (Ganondorf-1) [inline wiki/Help_centre/Upgrading_to_lazer/fr.md:47]:
```suggestion
| Ajustement de la difficulté des mods | ![Non][false] | ![Oui][true][^difficulty-adjust] |
```

File: wiki/Help_centre/Upgrading_to_lazer/fr.md [hunk 17]
```diff
@@ -0,0 +1,293 @@
...
+
+#### Pourquoi ne puis-je pas jouer avec un nombre illimité de FPS ?
+
+Au-delà d'un certain seuil, il n'y a aucune raison de fonctionner à des fréquences d'images plus élevées. Lazer utilise diverses nouvelles technologies pour s'assurer que la latence la plus faible possible soit atteinte sans nécessiter des fréquences d'images élevées. Cela continuera à s'améliorer à l'avenir, car il nous reste encore quelques améliorations à mettre en œuvre.
+
+Lazer prend en compte l'entrée à 1 000 Hz indépendamment du limiteur de FPS, c'est pourquoi le réglage maximal du limiteur s'arrêtera également à 1 000 FPS.
+
+Si vous êtes curieux de savoir comment cela affecte la latence d'entrée et de tester votre propre perception, veuillez exécuter le "certificateur de latence" intégré en bas des paramètres.
+
+Vous pouvez également [lire ce document technique] (https://github.com/ppy/osu/wiki/Latency-and-unlimited-frame-rates) qui explique la voie que nous empruntons et le raisonnement qu'elle implique.
```
Comment (Ganondorf-1) [inline wiki/Help_centre/Upgrading_to_lazer/fr.md:197]:
```suggestion
Vous pouvez également [lire ce document technique](https://github.com/ppy/osu/wiki/Latency-and-unlimited-frame-rates) qui explique la voie que nous empruntons et le raisonnement qu'elle implique.
```

**Review by Ganondorf-1 — CHANGES_REQUESTED** [review]

**Comment by The-Last-Cookie** [comment]
And there's another one: #8576

**Review by TicClick — APPROVED** [review]

**Pull request merged**